sops.go

```diff
@@ -288,6 +291,12 @@ type Metadata struct {
 	MessageAuthenticationCode string
 	Version                   string
 	KeySources                []KeySource
+	// Shamir specifies if the data key this file is encrypted with was
+	// split between all key sources using Shamir's Secret Sharing.
+	Shamir                    bool
```

Comment:
nit on the comment: "Shamir is `true` when the data key is split across multiple master keys according to the shamir's secret splitting algorithm"